tests/pytests/unit/modules/test_openscap.py

```diff
@@ -0,0 +1,376 @@
+import subprocess
+
+import pytest
+
+import salt.modules.openscap as openscap
+from tests.support.mock import MagicMock, Mock, patch
+
+random_temp_dir = "/tmp/unique-name"
+policy_file = "/usr/share/openscap/policy-file-xccdf.xml"
+
+
+class OpenscapTestMock:
+    def __init__(self):
+        import salt.modules.openscap
+
+        salt.modules.openscap.__salt__ = MagicMock()
+        self.patchers = [
+            patch("salt.modules.openscap.__salt__", MagicMock()),
+            patch("salt.modules.openscap.shutil.rmtree", Mock()),
+            patch(
+                "salt.modules.openscap.tempfile.mkdtemp",
+                Mock(return_value=random_temp_dir),
+            ),
+            patch("salt.modules.openscap.os.path.exists", Mock(return_value=True)),
+        ]
+        for patcher in self.patchers:
+            self.apply_patch(patcher)
+
+    def apply_patch(self, patcher):
+        patcher.start()
+
+    def close(self):
+        for patcher in self.patchers:
+            patcher.stop()
+
+
+@pytest.fixture
+def openscap_mocks():
+    mocks = OpenscapTestMock()
+    yield mocks
+    mocks.close()
+
+
+@pytest.fixture
+def configure_loader_modules(openscap_mocks):
+    return {openscap: {"__salt__": {"cp.push_dir": MagicMock()}}}
```

Comment:
```suggestion

@pytest.fixture
def random_temp_dir(tmp_path):
    tmp_dir = tmp_path / "unique"
    tmp_dir.mkdir()
    return str(tmp_dir)

@pytest.fixture
def configure_loader_modules(random_temp_dir):
    with patch("salt.modules.openscap.__salt__", MagicMock()),   patch("salt.modules.openscap.shutil.rmtree", Mock()), patch(
                "salt.modules.openscap.tempfile.mkdtemp",
                Mock(return_value=random_temp_dir),
            ), patch("salt.modules.openscap.os.path.exists", Mock(return_value=True)):
        yield {openscap: {"__salt__": {"cp.push_dir": MagicMock()}}}
```

Properly formatted, of course.

Reply:
Done